Submitted By:            Fernando de Oliveira <famobr at yahoo dot com dot br>
Date:                    2014-06-24
Initial Package Version: 1.36.4
Upstream Status:         Confirmed
Origin:                  Upstream
URL:                     https://bugzilla.gnome.org/show_bug.cgi?id=732122#c2
Description:             Revert "Update after API break in HarfBuzz": undefined reference to `hb_ot_layout_language_get_required_feature'.

--- a/pango/pango-ot-info.c.orig	2014-06-23 14:30:09.000000000 -0300
+++ b/pango/pango-ot-info.c	2014-06-24 10:22:36.777928910 -0300
@@ -189,11 +189,10 @@
 					   &l_index);
   if (language_index) *language_index = l_index;
 
-  hb_ot_layout_language_get_required_feature (info->hb_face, tt,
-                                              script_index,
-                                              l_index,
-                                              required_feature_index,
-                                              NULL);
+  hb_ot_layout_language_get_required_feature_index (info->hb_face, tt,
+                                                   script_index,
+                                                   l_index,
+                                                   required_feature_index);
 
   return ret;
 }
